ui/job-view/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+
+import * as notificationStore from './stores/notifications';
+
+export default () => {
+  const reducers = combineReducers({
```

Comment:
I went ahead and added ``combineReducers`` here.  I obviously don't need it now, but will for the next PR, so it'll make the churn less on the follow-up PRs for ``selectedJob``, etc.